packages/client/src/components/form/Button.tsx

```diff
@@ -57,7 +57,11 @@ export function ButtonField(props: ButtonFieldProps) {
     isCompleted
   }) => {
     if (isCompleted) {
-      setFieldTouched(fieldDefinition.name)
+      const fieldsHavingSameTrigger = fields.filter(
+        (f) => isFieldButton(f) && f.options.trigger === trigger.name
```

Comment:
In what kind of use case can you have two buttons triggering the same HTTP request? Can you share your form config json on this part? What happens if you do not set the buttons touched?

Reply:
Thanks for the explanation. Makes sense! Can you please add a comment above the code change to explain the 
>As we currently do not have any feature of making a conditionally required field, I had to duplicate the button with alternative expressions.

so next time I'm looking at the code and ask why was this made I remember :)

Reply:
Sure! I will add this comment for clarification. Thank you, Riku